packages/acpica.rb

```diff
@@ -0,0 +1,30 @@
+require 'package'
+
+class Acpica < Package
+  description 'ACPI tools, including Intel ACPI Source Language compiler'
+  homepage 'https://acpica.org/'
+  version '20200326'
+  source_url 'https://acpica.org/sites/acpica/files/acpica-unix2-20200326.tar.gz'
+  source_sha256 '7e0d38d088ff4b2114d08e76201e000091d5a4af40ce7275950759299ba10878'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  def self.build
+    system 'make'
+  end
+
+  def self.install
+    system "install -Dm755 generate/unix/bin/iasl #{CREW_DEST_PREFIX}/bin/iasl"
```

Comment:
You could do something like this to avoid having to explicitly include each line and the path to each binary:
```
Dir.chdir 'generate/unix/bin' do
  system "for f in \$(ls | xargs); do install -Dm755 \$f #{CREW_DEST_PREFIX}/bin/\$f; done"
end
```
3 lines instead of 9.